spec.html

```diff
@@ -17492,6 +17495,15 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-function-definitions-static-semantics-containsusestrict">
+      <h1>Static Semantics: ContainsUseStrict</h1>
+      <emu-see-also-para op="ContainsUseStrict"></emu-see-also-para>
+      <emu-grammar>FunctionBody : FunctionStatementList</emu-grammar>
+      <emu-alg>
+        1. If the DirectivePrologue of |FunctionStatementList| contains a Use Strict Directive, return *true*; otherwise, return *false*.
```

Comment:
I thought this was defined somewhere as *DirectivePrologue*, because it's used that way in 15.1.2, but apparently that's not the case.